groups/sig-node/groups.yaml

```diff
@@ -19,6 +19,19 @@ groups:
   # subproject artifacts being stored in a given staging project
   #
 
+  - email-id: sig-node-leads@kubernetes.io
+    name: sig-node-leads
+    description: |-
+      SIG Node chairs and technical leads comms
+    settings:
+      WhoCanPostMessage: "ANYONE_CAN_POST"
+      ReconcileMembers: "true"
+    owners:
+      - dawnchen@google.com
+      - decarr@redhat.com
+      - mpatel@redhat.com
+      - S.Kanzhelev@gmail.com
```

Comment:
```suggestion
      - s.kanzhelev@gmail.com
```

Reply:
Is there any practical difference?

Reply:
Two more instances of uppercase:

- Ritazh@microsoft.com
- Security@Ubuntu.com

Reply:
https://github.com/kubernetes/k8s.io/pull/5177

Reply:
Yes there's a difference.. it's specifically about what Google sees the email as because it will try to reconcile in a case sensitive way